<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-21T08:45:17 IST" name="Default suite" finished-at="2022-12-21T08:46:55 IST" duration-ms="97966">
    <groups>
    </groups>
    <test started-at="2022-12-21T08:45:17 IST" name="Default test" finished-at="2022-12-21T08:46:55 IST" duration-ms="97966">
      <class name="TestNG1.Practice1">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestNG1.Practice1@49d904ec]" started-at="2022-12-21T08:45:17 IST" name="setup" finished-at="2022-12-21T08:45:53 IST" duration-ms="35913" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login()[pri:1, instance:TestNG1.Practice1@49d904ec]" started-at="2022-12-21T08:45:53 IST" name="login" finished-at="2022-12-21T08:46:41 IST" duration-ms="47889" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Webpage3] but found [BrowserStack Login | Sign Into The Best Mobile & Browser Testing Tool]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Webpage3] but found [BrowserStack Login | Sign Into The Best Mobile & Browser Testing Tool]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestNG1.Practice1.login(Practice1.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="screenshot(org.testng.ITestResult)[pri:0, instance:TestNG1.Practice1@49d904ec]" started-at="2022-12-21T08:46:41 IST" name="screenshot" finished-at="2022-12-21T08:46:50 IST" duration-ms="8562" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=Practice1.login()[pri:1, instance:TestNG1.Practice1@49d904ec] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- screenshot -->
        <test-method is-config="true" signature="quit()[pri:0, instance:TestNG1.Practice1@49d904ec]" started-at="2022-12-21T08:46:50 IST" name="quit" finished-at="2022-12-21T08:46:55 IST" duration-ms="5343" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- TestNG1.Practice1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
